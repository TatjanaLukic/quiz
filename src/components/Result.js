import React from "react";

const Result = props => {
  const result = (props.score / 90) * 100;
  return (
    <div
      style={{
        fontSize: "50px",
        color: "white",
        padding: "40px 0",
        textAlign: "center"
      }}
    >
      Your score: {result.toFixed(2)} %
    </div>
  );
};

export default Result;
