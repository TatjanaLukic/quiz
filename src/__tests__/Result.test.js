import React from 'react';

import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Result from '../components/Result';

configure({adapter: new Adapter()});

describe('<Result/>',() => {

it("spanshot matches", () => {
    const tree = shallow(<Result />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});