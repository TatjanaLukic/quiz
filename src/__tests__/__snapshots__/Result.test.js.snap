// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Result/> spanshot matches 1`] = `
<div
  style={
    Object {
      "color": "white",
      "fontSize": "35px",
      "padding": "30px 0",
      "textAlign": "center",
    }
  }
>
  Score: 
  NaN
   %
</div>
`;
